sklearn/linear_model/_cd_fast.pyx

```diff
@@ -32,7 +32,9 @@ cdef enum:
     # Max value for our rand_r replacement (near the bottom).
     # We don't use RAND_MAX because it's different across platforms and
     # particularly tiny on Windows/MSVC.
-    RAND_R_MAX = 0x7FFFFFFF
+    # It corresponds to the maximum representable value for
+    # 32-bit signed integers (i.e. 2^31 - 1).
+    RAND_R_MAX = 2147483647
```

Comment:
Have you compares, if cd gives the same results?

Reply:
The doc tests are meant to ensure that. Like the example in the docstring of ``GraphicalLasso``, which uses ``enet_coordinate_descent``.

also:
```
In [4]: %%cython
   ...: cdef enum:
   ...:     R = 0x7FFFFFFF
   ...: print(R)
   ...:
2147483647
```